### DISCUSSION TITLE
То что стоило сделать давно

### DISCUSSION THREAD

**Discussion by aniastria** [octothorp-team/public-dobrograd#737] [Предложения]
Я **не понимаю** до сих пор почему у пищи модель либо просто непонятный пакет, либо модель китайской лапши. Это малость бьет по атмосфере, особенно когда чтобы понять что лежит тебе нужно либо зажимать G и ждать, либо брать эту вещь в инвентарь. Причем модели различной еды уже есть на сервере.

**Comment by KOwOMei** [comment]
Привет.

Это предложение писали ранее. Челог же ответил, что делать всей еде разные модели он не будет из-за проблем с кэшом моделей.
